server/routes/termdb.singlecellSamples.ts

```diff
@@ -94,7 +94,9 @@ function validateSamplesNative(S: SingleCellSamplesNative, ds: any) {
 	}
 	if (samples.length == 0) throw 'no sample with sc data'
 	// getter returns array of {sample:<samplename>, files:[]} where files is gdc specific. each sample is an obj and allows to add ds-specific stuff
-	S.get = () => samples
+	S.get = () => {
+		return { samples, fields: ['sample'], columnNames: ['Sample'] }
```

Comment:
by doing this, you're assuming a non-gdc dataset table should only have a Sample column

which will be wrong

this still should be defined in dataset file. otherwise for another non-gdc dataset we won't be able to add columns to sample table, without breaking existing one